### PR TITLE
OpenURL() for Android Platform

### PR DESCRIPTION
**Pull request by futureapricot**
This PR implements the OpenURL() function for Android Platform.
It will call an Intent to open the given url with the default browser app.

### CODE REVIEW

File: src/rcore.c [hunk 1]
```diff
@@ -3457,6 +3457,29 @@ void OpenURL(const char *url)
 #endif
 #if defined(PLATFORM_WEB)
         emscripten_run_script(TextFormat("window.open('%s', '_blank')", url));
+#endif
+#if defined(PLATFORM_ANDROID)
+        JNIEnv *env;
```
Comment (raysan5) [inline src/rcore.c:3462]: Please, could you init it to NULL?

File: src/rcore.c [hunk 2]
```diff
@@ -3457,6 +3457,29 @@ void OpenURL(const char *url)
...
+        JavaVM *vm = CORE.Android.app->activity->vm;
```
Comment (raysan5) [inline src/rcore.c:3463]: As per [the documentation](https://developer.android.com/training/articles/perf-jni#javavm-and-jnienv), `JNIEnv` and `JavaVM` require `jni.h`, shouldn't the include be added?

File: src/rcore.c [hunk 3]
```diff
@@ -3457,6 +3457,29 @@ void OpenURL(const char *url)
...
+        (*vm)->AttachCurrentThread(vm, &env, NULL);
+
+        jstring url_string = (*env)->NewStringUTF(env, url);
```
Comment (raysan5) [inline src/rcore.c:3466]: Please, could you follow [raylib naming conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md) when naming variables on raylib code? It should be `urlString`. Same for the following code.

File: src/rcore.c (hunk 2 above)
Comment (futureapricot) [reply src/rcore.c:3464]: You're right! somehow I missed it!

**Comment by futureapricot** [comment]
I have made the mentioned corrections

**Comment by raysan5** [comment]
@futureapricot Thanks for the improvement and the review

**Pull request merged**